azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -438,3 +555,298 @@ def show(self, name: str, instance_name: str, resource_group_name: str) -> dict:
 
     def list(self, instance_name: str, resource_group_name: str) -> Iterable[dict]:
         return self.ops.list_by_resource_group(resource_group_name=resource_group_name, instance_name=instance_name)
+
+    def _get_endpoint_host(self, endpoint_type: DataflowEndpointType, **kwargs) -> str:
```

Comment:
you can put in host, eventhub_namespace as optional params here

Reply:
same comment for the other helpers

Reply:
is using kwargs not ok here? I do see existing helper function elsewhere using kwargs